indico/modules/events/management/templates/event_protection.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'layout/base.html' %}
+
+{% from 'forms/_form.html' import form_header, form_footer, form_rows, form_fieldset %}
+
+{% block title %}{% trans %}Event protection{% endtrans %}{% endblock %}
+
+{% block content %}
+    {{ form_header(form, id='event-protection-form') }}
+    {{ form_rows(form, skip=form.priv_fields) }}
+    {% call form_fieldset(_("Session coordinator rights")) %}
+        <div class="action-box highlight">
+            <div class="section">
+                <div class="icon icon-info"></div>
+                <div class="text">
+                    <div class="label">{% trans %}Basic rights{% endtrans %}</div>
+                    <div>
+                        <ul>
+                            <li>{% trans %}Restricted session timetable management (add contributions, create breaks){% endtrans %}</li>
+                            <li>{% trans %}View comments from the conference manager (no modifications){% endtrans %}</li>
+                            <li>{% trans %}View contribution list (no modifications){% endtrans %}</li>
+                            <li>{% trans %}View session main data (no modifications){% endtrans %}</li>
```

Comment:
Not sure if this shouldn't be a big block instead. After all, it's the same message. Not sure how to deal with the list items, though...